README.md

```diff
@@ -1,161 +1,279 @@
-# hello-microverse
-This is just basic min-project to must github flow 
+<a name="readme-top"></a>
 
- 
- ![murple_logo](https://user-images.githubusercontent.com/110924992/218853728-1a8f5726-cd6a-4613-85ed-38dd5132cd9f.png)
- 
- 
- 
- 
 
- 
-Microverse README Template
-📗 Table of Contents
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-📖 About the Project
+  <h3><b>Hello Microverse</b></h3>
 
-🛠 Built With
-html
-& css
+</div>
 
-Tech Stack
+<!-- TABLE OF CONTENTS -->
 
-Key Features
+# 📗 Table of Contents
 
-🚀 Live Demo
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-💻 Getting Started
-Setup
+<!-- PROJECT DESCRIPTION -->
 
+# 📖 [Hello microverse] <a name="about-project"></a>
+this just a basic project and the 1st project on the module
+This help me having to understand about linters 
 
-Prerequisites
+**[Hello hello microverse]** 
 
-Install
+## 🛠 Built With <a name="built-with"></a>
 
-Usage
+### Tech Stack <a name="tech-stack"></a>
 
-Run tests
+> Describe the tech stack and include only the relevant sections that apply to your project.
 
-Deployment
+<details
+  <summary>HTML5</summary>
+  <ul>
+    <li>HTML Tutorial from W3school</li>
+  </ul>
+</details>
 
-👥 Authors
+<details>
+  <summary>CSS3</summary>
+  <ul>
+    <li>CSS3 Tutorial from W3school</li>
+  </ul>
+</details>
 
-🔭 Future Features
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li>Linters Tutorial</a></li>
+  </ul>
+</details>
 
-🤝 Contributing
+<!-- Features -->
 
-⭐️ Show your support
+### Key Features <a name="key-features"></a>
 
-🙏 Acknowledgements
+- **[html 5]**
+- **[css 3]**
+- **[linters]**
 
-❓ FAQ (OPTIONAL)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-📝 License
+<!-- LIVE DEMO -->
 
-📖 [your_project_name]
-Describe your project in 1 or 2 sentences.
+## 🚀 Live Demo <a name="live-demo"></a>
 
-[hello microvese] is a...
 
-🛠 Built With
-  .html
-    .css
-Tech Stack
-Describe the tech stack and include only the relevant sections that apply to your project.
 
-Client
 
-Server
-node.js
+> Add a link to your deployed project.
 
-Database
+- [Live Demo Link]()
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] kindly consider removing this link or note it as N/A as you are not deploying this project to avoid broken links which are not professional